app/models/article.rb

```diff
@@ -63,6 +65,10 @@ def article_date_cannot_be_in_the_future
     end
   end
 
+  def edit_summary
```

Comment:
Why this separate method for the summary? Can you not use the summary method itself?